wd_sched.c

```diff
@@ -429,6 +447,8 @@ int wd_sched_rr_instance(const struct wd_sched *sched, struct sched_params *para
 	sched_info[numa_id].ctx_region[mode][type].valid = true;
 	sched_info[numa_id].valid = true;
 
+	wd_sched_map_cpus_to_dev(sched_ctx);
```

Comment:
It is recommended that you test the scenario in which the scheduler is used to execute business on a multi-device platform. Your logic here will cause errors.